dbcon/joblist/primitivestep.h

```diff
@@ -1564,7 +1571,9 @@ class TupleBPS : public BatchPrimitive, public TupleDeliveryStep
       boost::scoped_array<uint8_t> largeNullMemory;
       boost::scoped_array<boost::shared_array<uint8_t>> smallNullMemory;
       uint32_t matchCount;
-
+      uint64_t memUsedByOutputRG = 0;
```

Comment:
Why don't we assign this member in the ctor?

Reply:
An argument can be made, but it's moot, since the variable is no longer used. I removed it.